build/common/test_list.txt

```diff
@@ -4,4 +4,4 @@ libs/math/test_math
 libs/image/test_image compare libs/image/tests/reference/
 libs/utils/test_utils
 tools/matc/test_matc
-tools/cmgen/test_cmgen compare
\ No newline at end of file
+tools/cmgen/test_cmgen compare
```

Comment:
Trailing newline is required here, the test wasn't actually running before this commit.